tests/test_unit.py

```diff
@@ -92,7 +92,7 @@ async def test_listener_start(listener, message, caplog):
 
         assert "Latest message telegram" in caplog.text
         assert "client is online on revolt" in caplog.text
-        assert "Frasier👂 on telegram:" in caplog.text
+        # assert "Frasier👂 on telegram:" in caplog.text
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **issue (testing):** Commented out assertion reduces test coverage.

This assertion seems to have been commented out. If it's no longer relevant due to changes in the implementation, consider removing it entirely or updating it to reflect the new expected behavior. If this is a temporary change, please add a TODO comment to clarify the intention.